src/Sarif.UnitTests/Readers/PropertyBagConverterTests.cs

```diff
@@ -5,11 +5,10 @@
 using System.Collections.Generic;
 using System.Linq;
 using FluentAssertions;
-using Microsoft.CodeAnalysis.Sarif.Readers;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Newtonsoft.Json;
 
-namespace Microsoft.CodeAnalysis.Sarif.Core
+namespace Microsoft.CodeAnalysis.Sarif.Readers
```

Comment:
Fixed namespace because I'd previously moved the file to a different folder but forgot to adjust namespace.